### ISSUE TITLE
Consider adding Gemfile, and docs for local build

### ISSUE THREAD

**Issue by yohell** [neicnordic/neic2017.nordforsk.org#26]
Bast please consider adding a `/Gemfile` to this and other similar repos:
```
source 'https://rubygems.org'
gem 'github-pages', group: :jekyll_plugins
```

this makes it so that `bundle install` works (cf below)

------------
If you use rbenv with ruby-build and ruby-bundler plugins, you can download the site source and all prerequisites, and serve it from your laptop:
```bash
$ eval "$(rbenv init -)" # Use rbenv if you don't already.
$ git clone git@github.com:neicnordic/neic2017.nordforsk.org.git # Get the sources
$ cd /neic2017.nordforsk.org
$ rbenv local 2.3.3 # This site (dir) uses this version of Ruby 
$ bundle install # Set up jekyll just the way GitHub has it.
$ jekyll serve # Pages now build and serve exactly like they will on GitHub. Happy hacking!
```

Here's an example of how you set up rbenv with ruby-build and ruby-bundler plugins on your laptop:
```bash
$ sudo apt install rbenv
$ git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
$ git clone git://github.com/carsomyr/rbenv-bundler.git ~/.rbenv/plugins/bundler
$ eval "$(rbenv init -)"
$ cd ~/.rbenv/plugins/ruby-build; git pull; cd - # optional: refresh list of available ruby versions.
$ rbenv install 2.3.3
$ rbenv global 2.3.3
$ gem install bundler # do this once for each new ruby version you install.
```

References:

* https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/
* http://dan.carley.co/blog/2012/02/07/rbenv-and-bundler/

**Comment by bast** [comment]
This looks like a very good documentation to me. I have never used this approach but I probably should. Can you copy paste this to the README.md and add such a Gemfile? I am asking this because you probably have such a setup already running. I would first need to set it up. Which I will eventually do.

**Comment by yohell** [comment]
Fixed in 29aa8aed435b341323a55ce8e0c194a410324aa7.
